@xen-orchestra/qcow2/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@xen-orchestra/qcow2",
+  "version": "0.0.0",
+  "description": "parse and produce qcow2 files",
+  "main": "dist/index.mjs",
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch",
+    "postversion": "npm publish --access public",
+    "clean": "rimraf dist/",
+    "test-integration": "tsc && node --test dist/consumer/ConsumerQcowStream.integ.mjs"
+  },
+  "author": "",
```

It seems that we are using this `author` value in our packages:
```suggestion
  "author": {
    "name": "Vates SAS",
    "url": "https://vates.fr"
  }
```